src/coordinate-grid-mixin.js

```diff
@@ -891,16 +898,24 @@ dc.coordinateGridMixin = function (_chart) {
         if (_chart.brushIsEmpty(extent)) {
             dc.events.trigger(function () {
                 _chart.filter(null);
-                _chart.redrawGroup();
+                redrawFn();
             }, dc.constants.EVENT_DELAY);
         } else {
             var rangedFilter = dc.filters.RangedFilter(extent[0], extent[1]);
```

Comment:
Hmm seems one thing is kinda strange with this... After applying a filter, a click outside the brush range intermittently does not clear the filter.  A console log reports that the new click generated the previous range, and then it falls into this code path.